NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Release Notes
+This file documents changes to Argus-frontend that are relevant for the users to know.
+
+## [Unreleased]
+
+### Fixed
+- Fix ARGUS logo clipping part of the name.
```

Comment:
I don't see this as a very important point for end users to know.

I still view the releases notes mostly as a "oh, btw, you need to do some manual action or change some config or change your workflow before/during/after an upgrade to this version")